.github/workflows/ci.yml

```
@@ -20,10 +20,10 @@ jobs:
     - name: Checkout repository
       uses: actions/checkout@v2
     - name: Install dependencies
-      run: npm install
+      run:  npm install --legacy-peer-deps
     - name: Run tests
       run: npm test
-    - if: matrix.node-version == 14
+     - if: matrix.node-version == 20
```

```suggestion
    - if: matrix.node-version == 20
```
Accidentally added an extra space here